Scripts/Mend SCA/get-all-users-under-global-org.py

```diff
@@ -0,0 +1,152 @@
+import requests
+import json
+import os
+import sys
+
+REQUEST_HEADERS = {
+        'Content-Type': 'application/json',
+        'Accept': 'application/json'
+}
+
+# Purpose: Get all of the users that are inside of a global organization
+# Requirements:
+#   Environment Variables: 
+#       WS_URL
```

Comment:
Please use MEND_EMAIL and MEND_URL when referring to email and urls that are formatted without /agent
WS_WSS_URL should be used for urls ending with [/agent ](https://docs.mend.io/bundle/unified_agent/page/getting_started_with_the_unified_agent.html#Setting-the-Minimum-Required-Configuration-Parameters)